### PR TITLE
Improve the command helpers and minor bug fixes

### PR DESCRIPTION
**Pull request by Praveenrajmani**

### CODE REVIEW

File: cmd/kubectl-directpv/clean.go [hunk 1]
```diff
@@ -35,65 +35,65 @@ import (
...
-	Use:           "release [VOLUME ...]",
+var cleanCmd = &cobra.Command{
+	Use:           "clean [VOLUME ...]",
 	Short:         "Cleanup stale volumes",
 	SilenceUsage:  true,
 	SilenceErrors: true,
 	Example: strings.ReplaceAll(
-		`# Release all stale volumes
-$ kubectl {PLUGIN_NAME} release --all
+		`1) Cleanup all stale volumes
```
Comment (balamurugana) [inline cmd/kubectl-directpv/clean.go:44]: Use `mc` style like `1. Cleanup ...`

File: cmd/kubectl-directpv/init_config_v1.go [hunk 2]
```diff
@@ -18,6 +18,10 @@ package main
 
 import directpvtypes "github.com/minio/directpv/pkg/apis/directpv.min.io/types"
 
+const (
+	selectValueYes = "yes"
```
Comment (balamurugana) [inline cmd/kubectl-directpv/init_config_v1.go:22]: if this contant is used at only one place, there is no need to declare

File: pkg/installer/args.go [hunk 3]
```diff
@@ -52,6 +52,7 @@ type Args struct {
 	csiProvisionerImage      string
 	nodeDriverRegistrarImage string
 	livenessProbeImage       string
+	Printer                  func(interface{})
```
Comment (balamurugana) [inline pkg/installer/args.go:55]: There is no need to output other than yaml format.

File: cmd/kubectl-directpv/utils.go [hunk 4]
```diff
@@ -127,3 +102,29 @@ func getInput(msg string) string {
...
+	}
+	printer = printYAML
+	if jsonOutput {
+		printer = printJSON
+	}
+	return nil
+}
+
+func dryRun() bool {
+	return dryRunFlag || yamlOutput || jsonOutput
```
Comment (balamurugana) [inline cmd/kubectl-directpv/utils.go:129]: Why does output mode control dry-run? Many commands would perform actual change and print results as per output mode.

File: pkg/installer/args.go (hunk 3 above)
Comment (Praveenrajmani) [reply pkg/installer/args.go:55]: installer will just print yaml

File: cmd/kubectl-directpv/utils.go (hunk 4 above)
Comment (Praveenrajmani) [reply cmd/kubectl-directpv/utils.go:129]: `kubectl directpv install -o yaml` is a dry-run mode

Comment (Praveenrajmani) [reply cmd/kubectl-directpv/utils.go:129]: we have removed `--dry-run` from install command @balamurugana

File: cmd/kubectl-directpv/init_config_v1.go (hunk 2 above)
Comment (Praveenrajmani) [reply cmd/kubectl-directpv/init_config_v1.go:22]: It is used in two places

**Review by balamurugana — APPROVED** [review]

**Pull request merged**